system/jlib/jptree.ipp

```diff
@@ -415,15 +416,20 @@ private:
     void replaceSelf(IPropertyTree *val);
 
 protected: // data
+    byte flags;
```

Comment:
Worth adding:
// pack with the link count
here as you have done for PTLocalIteratorBase ?